api/client/cluster/client.go

```diff
@@ -146,3 +147,44 @@ func (c *clusterClient) GetGossipState() *cluster.ClusterState {
 	}
 	return status
 }
+
+func (c *clusterClient) EnumerateAlerts(resource api.ResourceType) (*api.Alerts, error) {
```

Comment:
seems like there can be only one set of alert routes and pass resource type appropriate for volume or cluster alerts .. the code looks exactly the same except clusterPath and volumePath ?